app/pages/legal/TermsOfService.jsx

```diff
@@ -46,7 +44,7 @@ PLEASE READ THE FOLLOWING TERMS OF USE (“TERMS”) CAREFULLY BEFORE USING SF S
 
 6. Disclaimers
     1. SF Service Guide and all information contained in it are distributed and transmitted "as is" and “as available” without warranties of any kind, either express or implied, including but not limited to warranties of title or implied warranties of merchantability or fitness for a particular purpose with respect to the quality, content, accuracy, timeliness, completeness, currency, freedom from interruption, freedom from computer virus, freedom from errors or omissions, non-infringement of content placed in SF Service Guide including any of the design, information, text, graphics, images, pages, interfaces, links, software, or other information and items contained in or displayed in jps.
-    2. ShelterTech is not responsible for any special, indirect, incidental or consequential damages (regardless of whether they were foreseeable) that may arise from the use of, or the inability to use SF Service Guide, whether the materials contained in SF Service Guide are provided by sheltertech or a third party.
+    2. ShelterTechis not responsible for any special, indirect, incidental or consequential damages (regardless of whether they were foreseeable) that may arise from the use of, or the inability to use SF Service Guide, whether the materials contained in SF Service Guide are provided by sheltertech or a third party.
```

Comment:
Looks like you accidentally dropped a space here.

```suggestion
    2. ShelterTech is not responsible for any special, indirect, incidental or consequential damages (regardless of whether they were foreseeable) that may arise from the use of, or the inability to use SF Service Guide, whether the materials contained in SF Service Guide are provided by sheltertech or a third party.
```